pkg/results/report.go

```diff
@@ -83,7 +83,6 @@ func (o *Options) Reporter(spec *api.JobSpec, consoleHost string) (Reporter, err
 		Certificates: []tls.Certificate{cert},
 		RootCAs:      caCertPool,
 	}
-	tlsConfig.BuildNameToCertificate()
```

Comment:
Why?

Reply:
Because its deprecated and shouldn't be used: https://golang.org/pkg/crypto/tls/#Config.BuildNameToCertificate